auacm/app/static/html/editCompetition.html

```diff
@@ -1,18 +1,18 @@
-
-<div class="row">
-  <h1 class="pull-left">{{ title }}</h1>
-  <button type="button" class="btn btn-success pull-right margin-top col-md-2"
-          ng-click="createCompetition()" ng-show="isAdmin === 1">
-    Save
-  </button>
-</div>
-
-<div class="panel panel-default">
+<div class="margin-top panel panel-default">
   <div class="panel-heading">
-    <div class="panel-title">
-      Metadata
+    <div class="row">
+      <h1 class="panel-title col-md-10">{{ title }}</h1>
+      <div class="col-md-2 no-pad">
+        <button type="button" class="btn btn-success pull-right col-md-10"
+                ng-click="createCompetition()" ng-show="isAdmin === 1">
+          Save
+        </button>
+      </div>
     </div>
   </div>
+
+  <h3 class="col-md-12">Metadata</h3>
+
   <div class="panel-body">
 
     <form name="createCompetitionForm" class="form-horizontal">
```

Comment:
What do you think about putting this part of the form in a well (adding the `well` class is all it takes). It just seems that the subtitle gets mixed in with the fields visually.